### PR TITLE
Fixed code blocks and indenting cli_cache_subcommands.

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- Content fix or rewrite

## Summary

When this pull request is merged, it will...
Fix code blocks and indenting.

### CODE REVIEW

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md [hunk 1]
```diff
@@ -36,15 +36,15 @@ Magento 2 has the following {% glossarytooltip 0bc9c8bc-de1a-4a06-9c99-a89a29c30
...
+```bash
+   bin/magento cache:status
 ```
 
 <!-- where `--bootstrap=` is a URL-encoded associative array of Magento [application bootstrap parameters]({{ page.baseurl }}/config-guide/bootstrap/magento-how-to-set.html#config-bootparam-overview) and values. -->
 
 A sample follows:
 
-```terminal
+```bash
```
Comment (keharper) [inline guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]:
```suggestion
```terminal
```

Comment (keharper) [reply guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]: bash is for input, terminal is for output. 

Bash statements are rendered with a command line prompt.

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md [hunk 2]
```diff
@@ -73,9 +73,9 @@ You can clean (also referred to as _flush_ or _refresh_) cache types using eithe
 
 Command options:
 
-```terminal
-bin/magento cache:enable [type] ... [type]
-bin/magento cache:disable [type] ... [type]
+```bash
+   bin/magento cache:enable [type] ... [type]
+   bin/magento cache:disable [type] ... [type]
```
Comment (keharper) [inline guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]: Break lines 77 and 78 into separate command line examples.

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md [hunk 3]
```diff
@@ -84,22 +84,22 @@ Where omitting `[type]` enables or disables all cache types at the same time. Th
...
+   bin/magento cache:disable db_ddl full_page
 ```
 
 Sample result:
 
-```terminal
-Changed cache status:
-    db_ddl: 1 -> 0
- full_page: 1 -> 0
+```bash
```
Comment (keharper) [inline guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]:
```suggestion
```terminal
```

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md [hunk 4]
```diff
@@ -122,36 +122,36 @@ Flush cache types if you've already tried cleaning the cache and you're still ha
 
 Command usage:
 
-```terminal
-bin/magento cache:clean [type] ... [type]
-bin/magento cache:flush [type] ... [type]
+```bash
+   bin/magento cache:clean [type] ... [type]
```
Comment (keharper) [inline guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]: Break lines 126 and 127 into separate command line examples.

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md [hunk 5]
```diff
@@ -122,36 +122,36 @@ Flush cache types if you've already tried cleaning the cache and you're still ha
...
-compiled_config
-eav
-customer_notification
-config_integration
-config_integration_api
-full_page
-config_webservice
-translate
-vertex
+```bash
```
Comment (keharper) [inline guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]:
```suggestion
```terminal
```

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]: Done.

File: guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md (hunk 4 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md]: Done.

**Comment by dobooth** [comment]
running tests

**Pull request merged**